src/game/boe.actions.cpp

```diff
@@ -3310,4 +3310,15 @@ void handle_rename_pc() {
 		pick_pc_name(choice,nullptr);
 	put_pc_screen();
 	put_item_screen(stat_window);
+}
+
+void handle_menu_automap() {
```

Comment:
Is there a reason for this to be different from showing the automap via hotkey or button click?

Reply:
A good reason? Not sure. But as it is, through the menu, you get a buffer message if it doesn't open. Through the button, if it doesn't open, there's no message.

Reply:
That's… kinda weird. If it doesn't open for whatever reason, surely it should tell you why?

Reply:
I could make that change.